src/app/zap-templates/zcl/data-model/chip/camera-av-stream-management-cluster.xml

```diff
@@ -0,0 +1,576 @@
+    <cluster code="0x0551"/>
+    <cluster code="0x0552"/>
+    <item fieldId="0" name="X1" type="int16u" default="0"/>
+    <item fieldId="1" name="Y1" type="int16u" default="0"/>
+    <item fieldId="2" name="X2" type="int16u" default="0"/>
+    <item fieldId="3" name="Y2" type="int16u" default="0"/>
+  </struct>
+
+  <cluster>
+    <domain name="General"/>
+    <name>Camera AV Stream Management</name>
+    <code>0x0551</code>
+    <define>CAMERA_AV_STREAM_MANAGEMENT_CLUSTER</define>
+    <description>The Camera AV Stream Management cluster is used to allow clients to manage, control, and configure various audio, video, and snapshot streams on a camera.</description>
+    <client init="false" tick="false">true</client>
+    <features>
+      <feature bit="0" code="PRIV" name="Privacy" summary="Privacy supported">
+        <optionalConform/>
+      </feature>
+      <feature bit="1" code="SCTL" name="SensorControl" summary="Sensor control supported">
+        <optionalConform/>
+      </feature>
+      <feature bit="2" code="ADO" name="Audio" summary="Audio Stream supported">
+        <optionalConform/>
+      </feature>
+      <feature bit="3" code="ICTL" name="ImageControl" summary="Image control supported">
+        <optionalConform/>
+      </feature>
+      <feature bit="4" code="VDO" name="Video" summary="Video Stream supported">
+        <optionalConform/>
+      </feature>
+      <feature bit="5" code="SNP" name="Snapshot" summary="Snapshot stream supported">
+        <optionalConform/>
+      </feature>
+      <feature bit="6" code="STOR" name="LocalStorage" summary="Local Storage available">
+        <optionalConform/>
+      </feature>
+    </features>
+    <server init="false" tick="false">true</server>
+    <globalAttribute code="0xFFFD" side="either" value="1"/>
+    <attribute code="0x0000" side="server" define="MAX_CONCURRENT_VIDEO_ENCODERS" type="int8u" reportable="true" optional="true">
+      <description>MaxConcurrentVideoEncoders</description>
+      <access op="read" privilege="administer"/>
+    </attribute>
+    <attribute code="0x0001" side="server" define="MAX_ENCODED_PIXEL_RATE" type="int32u" reportable="true" optional="true">
+      <description>MaxEncodedPixelRate</description>
+      <access op="read" privilege="administer"/>
+    </attribute>
+    <attribute code="0x0002" side="server" define="VIDEO_SENSOR_PARAMS" type="VideoSensorParamsStruct" reportable="true" optional="true">
+      <description>VideoSensorParams</description>
+      <access op="read" privilege="administer"/>
+    </attribute>
+    <attribute code="0x0003" side="server" define="NIGHT_VISION_CAPABLE" type="boolean" reportable="true" optional="true">
+      <description>NightVisionCapable</description>
+      <access op="read" privilege="administer"/>
+    </attribute>
+    <attribute code="0x0004" side="server" define="MIN_VIEW_PORT_WIDTH" type="VideoResolutionStruct" reportable="true" optional="true">
```

Comment:
Spec issue: using VideoResolutionStruct as the type for MinViewPortWidth and MinViewPortHeight makes no sense to me....

Reply:
Fixed in spec and incorporated in xml. The 2 attributes are collapsed to 1 attribute MinViewport of type VideoResolutionStruct(containing width and height).